blog/_posts/2017-05-23-whats-new-scala-native.md

```diff
@@ -0,0 +1,260 @@
+
+  ```scala
+  import java.io.{DataInputStream, File, FileInputStream}
+  val fis = new FileInputStream(new File("hello.txt"))
+  val dis = new DataInputStream(fis)
+  println(dis.readLine())
+  ```
+
+* [@densh][@densh] added initial support for Scala's Futures in [#618][#618],
+  using an implementation similar to that of Scala.js, where Futures will be
+  completed after the `main` method is executed.
+
+  ```scala
+  import java.util.concurrent.Future
+  import scala.concurrent.ExecutionContext.Implicits.global
+
+  object Test {
+    def main(args: Array[String]): Unit = {
+      println("Start")
+      Future {
+        println("Hello from the Future")
+      }.foreach(_ => ())
+      println("End")
+    }
+  }
+  ```
+
+* Scala Native now supports pointer subtraction. This work has been
+  contributed by [@jonas][@jonas] in [#624][#624].
+
+  ```scala
+  val carr: Ptr[CChar] = toCString("abcdefg")
+  val cptr: Ptr[CChar] = string.strchr(carr, 'd')
+  println(cptr - carr) // prints '3'
+  ```
+
+* [@ekrich][@ekrich] extended Scala Native's implementation of `String` to
+  support `toUpperCase` and `toLowerCase` in [#573][#573].
+* The implementation of `java.lang.Character` was extended to support
+  `Character.Subset` and `Character.UnicodeBlock` by [@densh][@densh] in
+  [#651][#651].
+* [@asoltysik][@asoltysik] implemented support for system properties in
+  [#591][#591].
+
+  ```scala
+  println(System.getProperty("java.version"))   // prints '1.8'
+  println(System.getProperty("file.separator")) // '\' on Windows,
+                                                // '/' elsewhere
+  ```
+* Scala Native can now read environment variables using `System.getEnv`,
+  thanks to [@jonas][@jonas]' efforts in [#606][#606].
+* `stdin` and `stdout` can reliably be read from and written to, thanks to
+  [@fduraffourg][@fduraffourg] and [@densh][@densh] in [#622][#622] and
+  [#659][#659].
+
+### Bugfixes
+
+* A bugfix has been contributed by [@jonas][@jonas] to `String.replace`,
+  fixing one broken benchmark at the same time. His work can be found in
+  [#616][#616].
+* `System.nanoTime` was fixed by [@brad-rathke][@brad-rathke] in [#549][#549].
+* [@xuwei-k][@xuwei-k] fixed `nativeAvailableDependencies` which didn't work
+   in the `test` configuration in [#565][#565].
+
+### Improvements to the tooling and integration
+
+* [@MasseGuillaume][@MasseGuillaume] and [@densh][@densh]
+  worked on refactoring Scala Native's sbt plugin to make it more idiomatic in
+  [#568][#568] and [#630][#630], which lead to fixing [#562][#562].
+* Follow up fixes were contributed by [@jonas][@jonas], and include [#639][#639]
+  and [#653][#653].
+
+### Preparing for a better garbage collector
+
+In this release, [@LukasKellenberger][@LukasKellenberger] introduced in
+[#539][#539] a new setting in the sbt plugin that lets users select what
+implementation of the garbage collector should be used. Currently, BoehGC
+and no GC are supported.
```

Comment:
I'd put it a bit differently. "It lets you to completely disable the garbage collector."